File: teamscale-client/src/main/java/com/teamscale/client/TestForPrioritization.java
package com.teamscale.client;

/**
 * {@link TestDetails} with information about their partition. <br/>
 * Note that two instances are considered equal if the test details are equal.
 */
public class TestForPrioritization {

	/**
	 * The uniform path the test.
	 */
	public String uniformPath;

	/** The reason the test has been selected. */
	public String selectionReason;

}

File: teamscale-client/src/main/kotlin/com/teamscale/client/TestForPrioritization.kt
package com.teamscale.client

/**
 * [TestDetails] with information about their partition. <br></br>
 * Note that two instances are considered equal if the test details are equal.
 */
class TestForPrioritization {

    /**
     * The uniform path the test.
     */
    var uniformPath: String? = null

    /** The reason the test has been selected.  */
    var selectionReason: String? = null

}
